crates/matrix-sdk-base/src/client.rs

```diff
@@ -13,15 +13,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#[cfg(feature = "e2e-encryption")]
+use std::ops::Deref;
```

Comment:
This doesn't seem necessary, `Deref` is not referenced in any new code.

Reply:
Deref was already present I just had to move the Arc usage from
```
#[cfg(feature = "e2e-encryption")]
use std::{ops::Deref, sync::Arc};
```

to not depend on e2e-encryption, since now is used also to store the Observable, otherwise the non e2e-encyption tests would fail